core/src/components/list-header/list-header.ios.vars.scss

```diff
@@ -8,7 +8,7 @@
 $list-header-ios-padding-start:       $item-ios-padding-start !default;
 
 /// @prop - Font size of the list header
-$list-header-ios-font-size:           min(1.375rem, 56px) !default;
+$list-header-ios-font-size:           dynamic-font-max(22px, 2.55) !default;
```

Comment:
`22px * 2.55 = 56.1px` which may be causing the weird screenshot diffs from the max being `56px` previously